DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/mmhost/src/part_1_pointers.cpp

```diff
@@ -0,0 +1,91 @@
+#include <sycl/ext/intel/fpga_extensions.hpp>
+#include <sycl/sycl.hpp>
+
+#include "exception_handler.hpp"
+
+using namespace sycl;
+
+struct PointerIP{
+  int *x; 
+  int *y; 
+  int *z;
```

Comment:
Could we do this instead? Is there a reason to prefer one option over the other?
```suggestion
  int *x, *y, *z;
```

Reply:
Might be useful to further show the difference between the two kernels, but it can be changed for brevity. 
@tiwaria1 what are your thoughts here?

Reply:
separate lines makes it easy for users to diff. You should add a note to the readme or a comment to the source code encouraging the user to compare the source files to see the differences.